README.md

```diff
@@ -4,9 +4,9 @@
 
 <p align="center">
   <a href="https://github.com/ForgeRock">
-    <img src="https://www.forgerock.com/themes/custom/forgerock/images/fr-logo-horz-color.svg" alt="Logo">
+    <img src="https://cdn-docs.pingidentity.com/navbar/ping-logo-horizontal.svg" alt="Logo">
   </a>
-  <h2 align="center">ForgeRock SDK for Android</h2>
+  <h2 align="center">Ping SDK for Android</h2>
```

Comment:
I'm not sure if we should update this SDK to "Ping SDK". Or did I miss a discussion/decision about this? 
As far as I know, the "Ping SDK for Android" is the new SDK (we called it initially "Unified SDK"), located in a separate repo here...: https://github.com/ForgeRock/ping-android-sdk.